rennovate.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
typo on the filename? Unsure if renovate would still pick up on it.

Reply:
It is, ive done this before too. Ive convinced myself that renovate has two n